image/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM fedora
+
+ADD servicebroker /opt/servicebroker/servicebroker
+RUN chmod +x /opt/servicebroker/servicebroker
```

Comment:
Is this really necessary? Think considering we just built, this should be a executable already?

Reply:
good point